desktop/dashboard.md

```diff
@@ -227,23 +227,40 @@ To interact with remote repositories:
 
 ## Explore volumes
 
-The **Volumes** view in Docker Dashboard enables you to easily create and delete [volumes](../storage/volumes.md) and see which ones are being used. If you are a developer subscribed to a Pro or a Team plan, you can also see which container is using a specific volume and explore the files and folders in your volumes.
+A [volume](../storage/volumes.md) is a directory on a disk or in a container. Volumes are directly managed by Docker and are stored as part of the host file system.
+
+The **Volumes** view in Docker Dashboard enables you to easily create and delete volumes and see which ones are being used. If you are a developer subscribed to a Pro or a Team plan, you can also see which container is using a specific volume and explore the files and folders in your volumes.
 
 {% include upgrade-cta.html
   body="To start exploring the details in your volumes, upgrade to a Pro or a Team plan."
   header-text="This feature requires a Pro or a Team plan"
   target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_volumes"
 %}
 
+The following table lists the Volumes UI features available in each plan:
+
+{% assign yes = '![yes](/images/green-check.svg){: .inline style="height: 14px; margin: 0 auto"}' %}
+
+| Feature      | Free | Pro | Team   |
+|:-------------|:---------:|:----------:|:----------:|
+|  View, create, and delete volumes| {{ yes }}    | {{ yes }}   |  {{ yes }}    |
+| See which volumes are in use | {{ yes }} | {{ yes }}   |  {{ yes }}    |
+| See the size of the volume | {{ yes }}  | {{ yes }}   |  {{ yes }}    |
+| See volume data details (modified date, size, kind) | X | {{ yes }}   |  {{ yes }}    |
+| See which container is using a volume | X | {{ yes }}   |  {{ yes }}    |
+| Save or delete data from a volume | X| {{ yes }}   |  {{ yes }}    |
+
 ### Manage volumes
 
-By default, the **Volumes** view displays a list of all the volumes.
+By default, the **Volumes** view displays a list of all the volumes. Volumes that are currently used by a container display the **In Use** badge.
 
 ![List volumes](images/volumes-list.png){:width="700px"}
 
+Use the **Search** field to search for any specific volumes. You can also sort volumes by the name, the date created, and the size of the volume.
+
 To explore the details of a specific volume, select a volume from the list. This opens the detailed view.
 
-The **In Use** tab displays the name of the container that’s using the volume, the image name, the port number used by the container, CPU, memory, disk read/write, and network I/O consumed by the container.
+The **In Use** tab displays the name of the container using the volume, the image name, the port number used by the container, and the target. A target is a mountpoint inside a container that gives access to the file system on the host.
```

Comment:
```suggestion
The **In Use** tab displays the name of the container using the volume, the image name, the port number used by the container, and the target. A target is a path inside a container that gives access to the volume files.
```
(it’s not on the host, it’s inside the VM)

Reply:
Updated.